Utility.h

```diff
@@ -1,17 +1,19 @@
 #pragma once
 
+#include <string>
 #include <vector>
 #include <memory>
-#include "Item.h"
-
 
+struct Item;
 std::vector<std::unique_ptr<Item>> makeHelpfulItems(int num);
 
 std::vector<std::unique_ptr<Item>> makeDefensiveItems(int num);
 
+std::vector<std::unique_ptr<Item>> makeAttackItems(int num);
+
```

Comment:
Instructions did not say to add this
- [ ] REVISE

Reply:
done removed